packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/typescript/clearFunk10.yml

```diff
@@ -0,0 +1,22 @@
+languageId: typescript
+command:
+  version: 5
+  spokenForm: clear funk
+  action: {name: clearAndSetSelection}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: namedFunction}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: const foo = function *bar() {}
```

Comment:
It's actually quite atrocious that this syntax is valid :p
I guess we do the right thing here

Reply:
Haha yes this is a definite wtf 😄